### ISSUE TITLE
Logging should hide thread exit messages

### ISSUE THREAD

**Issue by darthwalsh** [OmniSharp/omnisharp-vscode#3184]
## Issue Description ##
The output of my ASP.NET Core 2.1 app is cluttered with output like warning messages like:
```
The thread 16212 has exited with code 0 (0x0).
The thread 27664 has exited with code 0 (0x0).
```

## Steps to Reproduce ##
Create a new aspnet core app, run it, make some requests, wait for a few minutes.

## Expected Behavior ##
If a thread exits successfully with code 0, by default don't show a warning message.

(Or in #1154 I see that @gregg-miskelly offers to add more knobs to launch.json, if by default devs want to see all these thread exit warnings.?)

## Actual Behavior ##
After running some requests and coming back to VS Code, the logs look like:

```
Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker:Information: Executed action Web.Controllers.ValuesController.Post (Web) in 195.9411ms
info: Microsoft.AspNetCore.Hosting.Internal.WebHost[2]
      Request finished in 328.0503ms 200 application/json; charset=utf-8
Microsoft.AspNetCore.Hosting.Internal.WebHost:Information: Request finished in 328.0503ms 200 application/json; charset=utf-8
The thread 27876 has exited with code 0 (0x0).
The thread 16212 has exited with code 0 (0x0).
The thread 27664 has exited with code 0 (0x0).
The thread 4980 has exited with code 0 (0x0).
The thread 13016 has exited with code 0 (0x0).
The thread 22004 has exited with code 0 (0x0).
The thread 22656 has exited with code 0 (0x0).
The thread 28564 has exited with code 0 (0x0).
The thread 11096 has exited with code 0 (0x0).
The thread 16292 has exited with code 0 (0x0).
The thread 23580 has exited with code 0 (0x0).
The thread 17444 has exited with code 0 (0x0).
The thread 28656 has exited with code 0 (0x0).
```

### OmniSharp log ###
```
Starting OmniSharp server at 7/19/2019, 11:33:24 PM
    Target: c:\code\ExpLang\ExpLang.sln

OmniSharp server started.
    Path: C:\Users\cwalsh\.vscode\extensions\ms-vscode.csharp-1.21.0\.omnisharp\1.34.0\OmniSharp.exe
    PID: 19148

[info]: OmniSharp.Stdio.Host
        Starting OmniSharp on Windows 6.2.9200.0 (x64)
[info]: OmniSharp.Services.DotNetCliService
        DotNetPath set to dotnet
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Located 3 MSBuild instance(s)
            1: Visual Studio Community 2017 15.9.28307.145 - "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin"
            2: Visual Studio Enterprise 2019 16.1.28922.388 - "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin"
            3: StandAlone 16.0 - "C:\Users\cwalsh\.vscode\extensions\ms-vscode.csharp-1.21.0\.omnisharp\1.34.0\.msbuild\Current\Bin"
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Registered MSBuild instance: Visual Studio Enterprise 2019 16.1.28922.388 - "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin"
[info]: OmniSharp.Cake.CakeProjectSystem
        Detecting Cake files in 'c:\code\ExpLang'.
[info]: OmniSharp.Cake.CakeProjectSystem
        Could not find any Cake files
[info]: OmniSharp.MSBuild.ProjectSystem
        Detecting projects in 'c:\code\ExpLang\ExpLang.sln'.
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'c:\code\ExpLang\Web\Web.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'c:\code\ExpLang\Engine\Engine.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'c:\code\ExpLang\EngineTests\EngineTests.csproj'
[info]: OmniSharp.Script.ScriptProjectSystem
        Detecting CSX files in 'c:\code\ExpLang'.
[info]: OmniSharp.Script.ScriptProjectSystem
        Could not find any CSX files
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.CSharpFormattingWorkspaceOptionsProvider, Order: 0
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: c:\code\ExpLang\Web\Web.csproj
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.RenameWorkspaceOptionsProvider, Order: 100
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.EditorConfigWorkspaceOptionsProvider, Order: 200
[info]: OmniSharp.WorkspaceInitializer
        Configuration finished.
[info]: OmniSharp.Stdio.Host
        Omnisharp server running using Stdio at location 'c:\code\ExpLang' on host 20048.
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'c:\code\ExpLang\Web\Web.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'c:\code\ExpLang\Web\Web.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: c:\code\ExpLang\Engine\Engine.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'c:\code\ExpLang\Engine\Engine.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'c:\code\ExpLang\Engine\Engine.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: c:\code\ExpLang\EngineTests\EngineTests.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'c:\code\ExpLang\EngineTests\EngineTests.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'c:\code\ExpLang\EngineTests\EngineTests.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: Web
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: Engine
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: EngineTests
[info]: OmniSharp.Roslyn.CSharp.Services.Refactoring.V2.CachingCodeFixProviderForProjects
        Skipping code fix provider 'Microsoft.CodeAnalysis.Analyzers.MetaAnalyzers.CodeFixes.ApplyDiagnosticAnalyzerAttributeFix' because it's language doesn't match 'C#'.
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'c:\code\ExpLang\Web\Web.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: c:\code\ExpLang\Web\Web.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'c:\code\ExpLang\Web\Web.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: Web
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'c:\code\ExpLang\Engine\Engine.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: c:\code\ExpLang\Engine\Engine.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'c:\code\ExpLang\Engine\Engine.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: Engine
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'c:\code\ExpLang\Web\Web.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: c:\code\ExpLang\Web\Web.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'c:\code\ExpLang\Web\Web.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: Web
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'c:\code\ExpLang\Web\Web.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: c:\code\ExpLang\Web\Web.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'c:\code\ExpLang\Web\Web.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: Web
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'c:\code\ExpLang\Web\Web.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: c:\code\ExpLang\Web\Web.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'c:\code\ExpLang\Web\Web.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: Web
```

### C# log ###
```
Installing C# dependencies...
Platform: win32, x86_64

Downloading package 'OmniSharp for Windows (.NET 4.6 / x64)' (33536 KB).................... Done!
Validating download...
Integrity Check succeeded.
Installing package 'OmniSharp for Windows (.NET 4.6 / x64)'

Downloading package '.NET Core Debugger (Windows / x64)' (47489 KB).................... Done!
Validating download...
Integrity Check succeeded.
Installing package '.NET Core Debugger (Windows / x64)'

Downloading package 'Razor Language Server (Windows / x64)' (60117 KB).................... Done!
Installing package 'Razor Language Server (Windows / x64)'

Finished
```

## Environment information ##

**VSCode version**: 1.36.1
**C# Extension**: 1.21.0

<details><summary>Dotnet Information</summary>
.NET Core SDK (reflecting any global.json):
 Version:   2.1.700
 Commit:    c2ef055a0f

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.700\

Host (useful for support):
  Version: 2.1.11
  Commit:  d6a5616240

.NET Core SDKs installed:
  1.0.0-preview1-002702 [C:\Program Files\dotnet\sdk]
  1.1.0 [C:\Program Files\dotnet\sdk]
  2.0.2 [C:\Program Files\dotnet\sdk]
  2.0.3 [C:\Program Files\dotnet\sdk]
  2.1.401 [C:\Program Files\dotnet\sdk]
  2.1.403 [C:\Program Files\dotnet\sdk]
  2.1.500 [C:\Program Files\dotnet\sdk]
  2.1.602 [C:\Program Files\dotnet\sdk]
  2.1.700 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.5 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.5 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 1.0.0-rc2-3002702 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 1.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 1.0.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 1.1.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|
|---|---|---|
|code-settings-sync|Shan|3.4.0|
|cpptools|ms-vscode|0.24.0|
|csharp|ms-vscode|1.21.0|
|debugger-for-chrome|msjsdiag|4.11.6|
|gitlens|eamodio|9.8.5|
|java|redhat|0.47.0|
|javadebugger|donjayamanne|0.1.5|
|language-haskell|justusadam|2.6.0|
|powershell|ms-vscode|2019.5.0|
|python|ms-python|2019.6.24221|
|tslint|eg2|1.0.44|
|vscode-eslint|dbaeumer|1.9.0|
|vscode-jasmine-test-adapter|hbenl|1.5.0|
|vscode-java-debug|vscjava|0.20.0|
|vscode-java-dependency|vscjava|0.5.1|
|vscode-java-pack|vscjava|0.7.1|
|vscode-java-test|vscjava|0.18.1|
|vscode-maven|vscjava|0.18.1|
|vscode-test-explorer|hbenl|2.13.0|
|vscodeintellicode|VisualStudioExptTeam|1.1.8|;

</details>

**Comment by gregg-miskelly** [comment]
To be clear - I wouldn't call this a "warning". 

But your point that the full VS default is maybe not the right default for VS Code is a good point.

**Comment by gregg-miskelly** [comment]
Actually, looks like we don't even have a knob for configuring thread exit messages...

**Comment by StephanThierry** [comment]
Any update on this issue? It's still open and the milestone is only 25% complete.

**Comment by gregg-miskelly** [comment]
Fixed in 1.21.
